db/schema.rb

```diff
@@ -0,0 +1,28 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# Note that this schema.rb definition is the authoritative source for your
+# database schema. If you need to create the application database on another
+# system, you should be using db:schema:load, not running all the migrations
+# from scratch. The latter is a flawed and unsustainable approach (the more migrations
+# you'll amass, the slower it'll run and the greater likelihood for issues).
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 20170112162706) do
+
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+  enable_extension "postgis"
```

Comment:
I'm a little concerned at what the underlying sql for this might look like. RDS has pretty specific setup steps for postgis that we'll have to adhere to. See https://github.com/matschaffer/safecastapi/blob/beanstalk-deployment/db/structure.rds.sql#L16-L34 for the SQL they recommend in their documentation.

afaict, The postgis extension in RDS isn't owned by the master user, so there are some limitations in how we can set it up.

If you have any thoughts on how to ensure this app is okay with that from the get-go I'd love to hear them.